tuftool/src/root.rs

```diff
@@ -352,10 +369,16 @@ impl Command {
             .get();
         let signature_count = signed_root.signed().signatures.len();
         if threshold > signature_count as u64 {
-            return Err(error::Error::SignatureRoot {
-                threshold,
-                signature_count,
-            });
+            // Return an error when the "force" flag wasn't set
+            if !force {
+                return Err(error::Error::SignatureRoot {
+                    threshold,
+                    signature_count,
+                });
+            }
+            // Print out a warning letting the user that the generated file isn't complient with
```

Comment:
This should probably say "letting the user *know* that the..."

Also - compliant is misspelled. :)